website/docs/getting-started/for-designers/partials/best-practices.md

```diff
@@ -1,71 +1,87 @@
+Figma is an incredibly versatile tool, but this versatility can sometimes backfire leading to disorganization, inconsistency, and difficulty in handoffs and collaboration. These best practices are geared towards using Helios components and foundations effectively to alleviate some of these challenges. For documentation around broader Figma best practices within HashiCorp design teams, reach out to the HashiCorp [Design Ops team](https://sites.google.com/hashicorp.com/designknowledgehub/design-system) or visit the [#design-ops slack channel](https://hashicorp.slack.com/archives/C029GL8GJDV).
 
-!!! Info
+## Component availability
 
-These best practices have been curated by the Design Systems Team, Figma has its own set of [best practices](https://www.figma.com/best-practices/) for using the tool effectively.
+**Structure** is the legacy set of components and foundations used to connect HashiCorp products and consists Ember components and foundational styles paired with a limited set of Figma counterparts. **Structure is no longer supported and is in the process of deprecation and sunsetting.**
 
-!!!
+As we work to achieve parity between Structure and Helios, use this decision tree to understand how to move forward within your project:
 
-**Note:** Much of this already exists from Design Ops, waiting to collaborate with them on the content.
+![Decision tree flow chart](/assets/getting-started/designers/hds-decision-tree.png)
 
-## File hygiene
+**Does the element (or a comparable alternative) exist in Helios?**
 
-- Use a cover photo
-- Differentiate between delivered work, and work in progress
+If so, great! Using this element ensures it will be supported in future versions of Helios and that it shares a common visual language with the rest of the elements.
 
-### Naming conventions
+**If no, does the element (or a comparable alternative) exist in Structure?**
 
-Naming things is hard, and while naming components and elements within a design system is often different from naming elements further down the UX funnel, the same best practices can be applied.
+If yes, use this component with the understanding that:
 
-!!! Insight
+1. Structure will be deprecated/sunsetted in the future
+2. Not all Figma components are built in code
+3. There is no longer a support structure in place if errors/bugs arise
 
-Whatever method you choose to use, the most important thing is maintaining a consistent naming strategy across projects, files, and components.
+If no, or having future support is a must: design and build your own local element using Helios foundations (color, typography, elevation, etc) as a starting point.
+
+### Local components
+
+Components and patterns that are unique to your product should be designed and built locally, extending Helios components and foundations.
```

Comment:
Can we beef up this section a bit? What constitutes a "unique" component or pattern? What do you mean by built locally? What would you tell a designer who's never used a design system before?

Reply:
Ahh, what is in the insight banner below is what I was looking for here. What if you moved the content in that banner out of the banner and as a part of this section instead? That would flow a bit better.